site/docs/tutorial/ios-app.md

```diff
@@ -40,9 +40,9 @@ its dependencies.
 
 ### Install Xcode
 
-Download and install [Xcode](https://developer.apple.com/xcode/downloads/). The
-Xcode download contains the iOS libraries, the Objective-C compiler, and other
-tools required by Bazel to build iOS apps.
+Download and install [Xcode](https://developer.apple.com/xcode/downloads/).
+Xcode contains the compilers, SDKs, and other tools required by Bazel to build 
+apps for all of Apples platforms (including iOS).
```

Comment:
"...by Bazel to build Apple applications." Let's keep the wording simple.

Reply:
Fixed